src/config/graphical-session/fonts.md

```diff
@@ -1,5 +1,8 @@
 # Fonts
 
+[xbps-query(1)](https://man.voidlinux.org/xbps-query.1) can be used to list
```

Comment:
I wanted to link to https://docs.voidlinux.org/xbps/index.html#finding-files-and-packages here but I found many more links that referenced man pages so went with that instead. Going to this page would present the user with an immediate code block to do exactly what they'd be looking for at this point, so I don't know that I prefer the man page. (Should I change it?)

Reply:
link the docs page